assets/frontend/app.js

```diff
@@ -1,12 +1,13 @@
-(function () {
-  'use strict';
+(function() {
+    'use strict';
 
-  angular
-    .module('portal', [
-      'ui.router',
-      'invoices.portal'
-    ])
-    .constant('TEMPLATE_URL', '/static/frontend/templates/')
-  ;
+    angular
+        .module('portal', [
+            'ui.router',
+            'invoices.portal',
+            'users.portal',
+        ])
+        .constant('TEMPLATE_URL', '/static/frontend/templates/')
+        .constant('API_URL', '/api/users/');
```

Comment:
wrong. this means that if you use API_URL in the invoice then it will become /api/users/invoices/